src/java.base/share/classes/java/lang/StackTraceElement.java

```diff
@@ -355,27 +354,46 @@ public boolean isNativeMethod() {
      * @revised 9
      * @see    Throwable#printStackTrace()
      */
+    @Override
     public String toString() {
-        String s = "";
-        if (!dropClassLoaderName() && classLoaderName != null &&
-                !classLoaderName.isEmpty()) {
-            s += classLoaderName + "/";
+        int estimatedLength = Objects.requireNonNullElse(classLoaderName, "").length() + 1
+                + Objects.requireNonNullElse(moduleName, "").length() + 1
+                + Objects.requireNonNullElse(moduleVersion, "").length() + 1
+                + declaringClass.length() + 1
+                + methodName.length() + 1
+                + Math.max("Unknown Source".length(), Objects.requireNonNullElse(fileName, "").length()) + 1
+                + Integer.stringSize(lineNumber) + 1;
```

Comment:
`Integer.stringSize(lineNumber)` can be further simplify to `11`

Reply:
inlined as 12 (combined with `1` for trailing `)`)